.github/workflows/bokeh-ci.yml

```diff
@@ -139,76 +69,43 @@ jobs:
           key: ${{ runner.os }}-node-${{ hashFiles('bokehjs/package-lock.json') }}
 
       - name: Install node modules
-        run: |
-          pushd bokehjs
-          npm install -g npm
-          npm ci --no-progress
-          popd
+        shell: bash -l {0}
+        run: bash ci/install_node_modules.sh
 
       # Don't cache sampledata - want to test at least one real download
 
       - name: Install sampledata
-        shell: bash
-        run: |
-          python -c 'import bokeh; bokeh.sampledata.download(progress=False)'
+        shell: bash -l {0}
+        run: python -c 'import bokeh; bokeh.sampledata.download(progress=False)'
 
-      - name: Run bokehjs codebase checks
-        working-directory: ./bokehjs
-        shell: bash
-        run: |
-          node make build:all lint
+      # - name: Run bokehjs codebase checks
+      #   working-directory: ./bokehjs
+      #   shell: bash -l {0}
+      #   run: |
+      #     node make build:all lint
```

Comment:
@mattpap what you do you want to do about this in the immediate term:

* make codebase depend on build?
* remove here / move to BokehJS-CI?
* something else?

Reply:
I will answer tomorrow.